http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -4244,6 +4244,8 @@ components:
           description: region of app
         kubernetesRuntimeConfig:
           $ref: '#/components/schemas/KubernetesRuntimeConfig'
+        autopilot:
```

Comment:
It might be good to put a link to the Autopilot documentation in the swagger page, especially because this is GKE specific